### PR TITLE
Change Updatenotification

### PR DESCRIPTION
**Pull request by enoch85**
Follow up on: https://github.com/nextcloud/vm/commit/7c7b5f59473da6fd37ec7576abbbb7f22010a37d

@szaimen Any thoughts on this? I ask becasue you implemented it.

My idea here is to run updates less frequently. Once per month is enough.

### CODE REVIEW

**Comment by enoch85** [comment]
> Hm... Is there a specific reason why the update script should only run once a month and not every week? This means some instances might be vulnerable in worst case for a whole month without any updates installed...

Yes, it's about uptime. I'm aware of the downside, but if you want to update more frequently then you can just change the crontab. Once per month is enough from a security standpoint if you ask me. In worst case, some CVE might not be patched instantly (aka next week) but 3 weeks more is not a problem I'd say - maybe if you're a government, but not a "regular user".

File: nextcloud_install_production.sh [hunk 1]
```diff
@@ -564,7 +564,7 @@ nextcloud_occ config:app:set updatenotification notify_groups --value="[]"
 print_text_in_color "$ICyan" "Configuring update notifications specific for this server..."
 download_script STATIC updatenotification
 check_command chmod +x "$SCRIPTS"/updatenotification.sh
-crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * * * $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
+crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * 1-12 6 $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
```
Comment (szaimen) [inline nextcloud_install_production.sh:567]: I would adjust this. The logic is there to only send the notification once even though the acript runs daily.

File: nextcloud_update.sh [hunk 2]
```diff
@@ -794,6 +794,8 @@ if [ -f "$SCRIPTS"/updatenotification.sh ]
 then
     download_script STATIC updatenotification
     chmod +x "$SCRIPTS"/updatenotification.sh
+    crontab -u root -l | grep -v "$SCRIPTS/updatenotification.sh" | crontab -u root -
```
Comment (szaimen) [inline nextcloud_update.sh:797]: I wouldn adjust this

File: nextcloud_update.sh [hunk 3]
```diff
@@ -794,6 +794,8 @@ if [ -f "$SCRIPTS"/updatenotification.sh ]
...
+    crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * * 5 $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
```
Comment (szaimen) [inline nextcloud_update.sh:798]: Same

File: static/updatenotification.sh [hunk 4]
```diff
@@ -56,7 +56,7 @@ then
         notify_admin_gui \
         "New minor Nextcloud Update!" \
         "Nextcloud $NCMIN just became available. Since you are running Automatic \
-Updates on Saturdays at $AUT_UPDATES_TIME:00, you don't need to bother about updating \
+Updates every month on Saturdays at $AUT_UPDATES_TIME:00, you don't need to bother about updating \
```
Comment (szaimen) [inline static/updatenotification.sh:59]: Looks good

**Review by szaimen — CHANGES_REQUESTED** [review]

**Review by szaimen — COMMENTED** [review]
So if you really want to change it to monthly adjusting only the wording shoupd be enough

**Pull request merged**